examples/webgl_loader_kmz.html

```diff
@@ -55,7 +53,7 @@
 
 				scene.add( camera );
 
-				const grid = new THREE.GridHelper( 50, 50, 0xffffff, 0x333333 );
+				const grid = new THREE.GridHelper( 50, 50, 0xffffff, 0x7b7b7b );
```

Comment:
The default color values of `GridHelper` are `0x444444` and `0x888888` right now. Would it make sense to update these in the ctor of `GridHelper` as well?

Reply:
Good question! Replied in main thread, since it is an important question and I don't want to lose the discussion.